src/governance/classes.cpp

```diff
@@ -496,14 +496,24 @@ CAmount CSuperblock::GetPaymentsLimit(int nBlockHeight)
 
     const CBlockIndex* tipIndex = ::ChainActive().Tip();
     bool fMNRewardReallocated = llmq::utils::IsMNRewardReallocationActive(tipIndex);
-    bool fV20Active = llmq::utils::IsV20Active(tipIndex);
     if (!fMNRewardReallocated && nBlockHeight > tipIndex->nHeight) {
         // If fMNRewardReallocated isn't active yet and nBlockHeight refers to a future SuperBlock
         // then we need to check if the fork is locked_in and see if it will be active by the time of the future SuperBlock
         if (llmq::utils::GetMNRewardReallocationState(tipIndex) == ThresholdState::LOCKED_IN) {
             int activation_height = llmq::utils::GetMNRewardReallocationSince(tipIndex) + static_cast<int>(Params().GetConsensus().vDeployments[Consensus::DEPLOYMENT_MN_RR].nWindowSize);
             if (nBlockHeight >= activation_height) {
                 fMNRewardReallocated = true;
+            }
+        }
+    }
+
+    bool fV20Active = llmq::utils::IsV20Active(tipIndex);
```

Comment:
Not important, `fV20Active` check should be placed first before `fMNRewardReallocated` check: as v20 will be active first